.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ attach-to-workspace: &attach-to-workspace
     at: .
 
 system-builder-ruby31: &system-builder-ruby31
-  image: quay.io/3scale/system-builder:stream8-ruby3.1
+  image: quay.io/3scale/system-builder:stream8-ruby3.1-chrome126
```

Comment:
127+ didn't work with `--disable-gpu`? That's worrying.

Reply:
Yes it does. I was trying to fix current Circleci failures by downgrading chromedriver, with no success.